### ISSUE TITLE
Expand the Glossary with additional items

### ISSUE THREAD

**Issue by kapetan3sid** [polkadot-developers/polkadot-docs#243]
Acceptance criteria:
- add: PolkaVM
- System to auto-link
- System to list references to each glossary
   - Remove unused ones
   - Executor

**Comment by UtkarshBhardwaj007** [comment]
@nhussein11 @kapetan3sid  This task is not clear to me. Could you please add a description?

**Comment by nhussein11** [comment]
> [@nhussein11](https://github.com/nhussein11) [@kapetan3sid](https://github.com/kapetan3sid) This task is not clear to me. Could you please add a description?

@UtkarshBhardwaj007 Yes, the idea would be to add some terminology associated with the PolkaVM to the current glossary. All the related content is in this branch: [staging-ah-smart-contracts](https://github.com/polkadot-developers/polkadot-docs/tree/staging-ah-smart-contracts ) 

Our current glossary is here: https://docs.polkadot.com/polkadot-protocol/glossary/

Regarding the "system to auto link" and the "system to list references for each glossary," those were requests from @kianenigma in the following feedback video https://www.loom.com/share/069f845df68949c6a6d517208eea2082

Maybe it would be worth leaving those systems or bots for separate PRs so we can revisit it in detail, wdyt?

**Comment by nhussein11** [comment]
> [@nhussein11](https://github.com/nhussein11) [@kapetan3sid](https://github.com/kapetan3sid) This task is not clear to me. Could you please add a description?

@UtkarshBhardwaj007 Yes, the idea would be to add some terminology associated with the PolkaVM to the current glossary. All the related content is in this branch: [staging-ah-smart-contracts](https://github.com/polkadot-developers/polkadot-docs/tree/staging-ah-smart-contracts ) 

Our current glossary is here: https://docs.polkadot.com/polkadot-protocol/glossary/

Regarding the "system to auto link" and the "system to list references for each glossary," those were requests from @kianenigma in the following feedback https://hackmd.io/G9pVpfUdRPWGHDDeU0KUWg?both and he mentioned that here:
https://www.loom.com/share/8d837339b2d048d4a1b5b3ceafdd6d8b (min 20:43)

**Comment by kianenigma** [comment]
Building the system that I described might be a bit complicated, let's first brainstorm how much effort it takes, and if it is worth it. As noted in meetings, most of our efforts until the contracts launch should be on contracts. 

The main big exception here would be making things test-able.
